smoke/integer/a.rb

```diff
@@ -0,0 +1,21 @@
+integer_1 = Integer(1)
+# !expects NoMethodError: type=::Integer, method=foo
+integer_1.foo
+
+integer_2 = Integer("1")
+# !expects NoMethodError: type=::Integer, method=foo
+integer_2.foo
+
+integer_3 = Integer(Time.new)
+# !expects NoMethodError: type=::Integer, method=foo
+integer_3.foo
+
+integer_4 = Integer("10", 2)
+# !expects NoMethodError: type=::Integer, method=foo
+integer_4.foo
+
+# !expects ArgumentTypeMismatch: receiver=::Object, expected=(::Numeric | ::String | ::_ToI), actual=::Object
+Integer(Object.new)
+
+# !expects ArgumentTypeMismatch: receiver=::Object, expected=(::Numeric | ::String | ::_ToI), actual=nil
+Integer(nil)
```

Comment:
I don't know that `smoke/integer/*.rb` is correct as smoke tests for `Kernle#.Integer`. 😓 

Please tell me if there are other good locations. 🙏